pip/setuptools-rust.file

```diff
@@ -0,0 +1 @@
+Requires: rust py3-setuptools-scm py3-semantic-version py3-toml py3-typing-extensions
```

Comment:
remove rust from `Requires` and add `BuildRequires: rust` so that we do not deploy it with cmssw. We only need it at build time

Reply:
by the way, is `rust` a deps on this or `crytography`? I think it is needed by  `crytography`  so better to add `BuildRequires: rust` in  `crytography.file`

Reply:
crytography needs setuptools-rust, which needs rust.

Reply:
ok. I just wanted to see if we can avoid distributing rust along with cmssw. May be then just add
```
BuildRequires: rust
```
for both `setuptools-rust.file` and `crytography.file`. This way rust will not be deployed along with cmssw

Reply:
Sure.